src/test/kotlin/com/vacuumlabs/example/ExampleApplicationTests.kt

```diff
@@ -50,15 +53,17 @@ class ExampleApplicationTests @Autowired constructor(
     }
 
     @Test
+    @WithMockUser
```

Comment:
maybe add a test case without mock user to check for auth error?

Reply:
tests added 👍